src/java/org/apache/cassandra/index/IndexRegistry.java

```diff
@@ -94,6 +94,12 @@ public Index getIndex(IndexMetadata indexMetadata)
             return null;
         }
 
+        @Override
+        public Index.Group getIndexGroup(Index.Group.Key groupKey)
+        {
+            return Index.Group.EMPTY;
+        }
+
```

Comment:
I think the `IndexRegistry#getIndexGroup` methods don't have any usage now, all calls are to `SecondaryIndexManager#getIndexGroup`.

Reply:
ok, removed